pkg/cclient/README.md

```diff
@@ -0,0 +1,36 @@
+# Compiling the C Library
+
+## Creating the .h and the .so files:
+
+
```

Comment:
only need one line not 2 - I think this should be flagged by the markdown linter